packages/hint-x-content-type-options/src/hint.ts

```diff
@@ -39,6 +39,34 @@ export default class XContentTypeOptionsHint implements IHint {
 
     public constructor(context: HintContext) {
 
+        const isHeaderRequired = (element: IAsyncHTMLElement): boolean => {
+            if (!element) {
+                return false;
+            }
+
+            const nodeName = normalizeString(element.nodeName);
+
+            /*
+             * See:
+             *
+             *  * https://github.com/whatwg/fetch/issues/395
+             *  * https://fetch.spec.whatwg.org/#x-content-type-options-header
+             */
+
+            if (nodeName === 'script') {
+                return true;
+
+            }
+
+            if (nodeName === 'link') {
+                const relValues = (normalizeString(element.getAttribute('rel'), '')).split(' ');
```

Comment:
I have a question, is the `split` necessary?

Reply:
@sarvaje Yes, because of cases such as `<link rel="  alternate stylesheet " href="test.css">'`

Reply:
but the result will be the same if you do a ' alternate stylesheet '.includes('stylesheet'), without splitting it.

Reply:
@sarvaje 'stylesheetx '.includes('stylesheet') => `true` (should be `false`)

Reply:
ok!